localstack/services/apigateway/apigateway_listener.py

```diff
@@ -510,7 +511,7 @@ def get_lambda_event_request_context(method, path, data, headers, integration_ur
         # adding stage to the request context path.
         # https://github.com/localstack/localstack/issues/2210
         'path': '/' + stage + relative_path,
-        'resourcePath': relative_path,
+        'resourcePath': resource_path,
```

Comment:
nitpick: To make this backward compatible (as this function may be used from other places), I'd suggest that we add `resource_path` as an additional (optional) keyword argument:
```
def get_lambda_event_request_context(..., resource_path=None)
```
... and then fall back to `relative_path` if `resource_path` is not defined:
```
resource_path = resource_path or relative_path
...
```

Reply:
Sure, but I haven't found any other usages other than `#invoke_rest_api_integration()` 🤔